tools/run_tests/helper_scripts/build_python.sh

```diff
@@ -56,6 +56,12 @@ function is_linux() {
   fi
 }
 
+function inside-venv() {
+  if [ "${VIRTUAL_ENV}" ]; then
```

Comment:
https://google.github.io/styleguide/shell.xml?showone=Test,_%5B_and_%5B%5B#Test,_%5B_and_%5B%5B

Reply:
Should this be `[[ -n "${VIRTUAL_ENV}" ]]`